raw_metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+    "4.1.0-rc.7+amd64": {
+        "io.openshift.upgrades.graph.previous.add": "4.1.0-rc.6"
+    },
+    "4.1.0-rc.6+amd64": {
+        "io.openshift.upgrades.graph.previous.add": "4.1.0-rc.5"
+    },
+    "4.1.0-rc.8+amd64": {
+        "io.openshift.upgrades.graph.previous.add": "4.1.0-rc.7"
+    },
+    "4.1.0-rc.9+amd64": {
+        "io.openshift.upgrades.graph.previous.add": "4.1.0-rc.7,4.1.0-rc.8"
+    },
+    "4.1.0+amd64": {
+        "io.openshift.upgrades.graph.previous.add": "4.1.0-rc.4,4.1.0-rc.9"
+    },
+    "4.2.13-s390x+s390x": {
+        "io.openshift.upgrades.graph.previous.add": "4.2.10-s390x,4.2.12-s390x"
```

Comment:
We can drop this from this PR and from Quay labels, because 4.2.13-s390x has [blocked all incoming edges][1].

[1]: https://github.com/openshift/cincinnati-graph-data/blob/04c5b057d2fbc8d235d86a32b46cb4dda891023c/blocked-edges/4.2.13-s390x.yaml#L3

Reply:
@jwforres [removed the label][1], so we can drop this here.

[1]: https://github.com/wking/cincinnati-graph-data/commit/07ffd741320d25026340e5f43367e4f56ec0e177